### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by shrestharajatko**
Add Azure integration

### CODE REVIEW

**Review by KOTechnologies — APPROVED** [review]

File: azure.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	var out AccountsAzureInfoResp
+	err := client.RequestResponse("GET", "/azure/account", nil, &out)
+	{
+		if err != nil {
+			return nil, err
+		}
+	}
+	if len(out.AccountsAzureInfo) == 0 {
+		return nil, fmt.Errorf("Cloudcraft AWS Account Integrations not found or there are none")
+	}
```
Comment (vijaykotechnologies) [inline azure.go:38]: Needs to say Azure

File: azure.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+// AccountIntegrationAzure struct represents the data of an Account.
+type AccountIntegrationAzure struct {
+	ID         		*string `json:"id,omitempty"`
+	Name 			*string `json:"name,omitempty"`
+    ApplicationId 	*string `json:"applicationId,omitempty"`
+    DirectoryId 	*string `json:"directoryId,omitempty"`
+    SubscriptionId 	*string `json:"subscriptionId,omitempty"`
+    ClientSecret 	*string `json:"clientSecret,omitempty"`
+	ExternalID 		*string `json:"externalId,omitempty"`
```
Comment (vijaykotechnologies) [inline azure.go:15]: 4 lines above need formatting

File: examples/integrationaws/mainazure.go
Comment (vijaykotechnologies) [inline examples/integrationaws/mainazure.go:1]: This file needs to move in to a folder called integrationzure

File: azure.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+			AccountIntegrationAzureItem = accounts[i]
+			//ID := *accounts[i].ID
+			//Name := *accounts[i].Name
+			//log.Printf("ID %s, Name %s", ID, Name)
+			return AccountIntegrationAzureItem, nil
+		}
+	}
+	if (AccountIntegrationAzure{}) == AccountIntegrationAzureItem {
+		return AccountIntegrationAzureItem, fmt.Errorf("Cloudcraft AWS Account Integration not found")
+	}
```
Comment (vijaykotechnologies) [inline azure.go:64]: AWS --> Azure

File: azure.go [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+	return AccountIntegrationAzureItem, nil
+}
+
+// AccountIntegrationAzureCreate updates AWS Account Integration.
+func (client *Client) AccountIntegrationAzureCreate(acc *AccountIntegrationAzure) error {
```
Comment (vijaykotechnologies) [inline azure.go:69]: AWS --> Azure

File: azure.go [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+	return client.RequestResponse("POST", "/azure/account", acc, &acc)
+}
+
+// AccountIntegrationAzureUpdate updates AWS Account Integrations.
+func (client *Client) AccountIntegrationAzureUpdate(acc *AccountIntegrationAzure) error {
+	return client.RequestResponse("PUT", fmt.Sprintf("/azure/account/%v", *acc.ID), acc, &acc)
+}
+
+// AccountAzureDelAccountIntegrationAzureDeleteete updates AWS Account Integrations.
+func (client *Client) AccountIntegrationAzureDelete(acc *AccountIntegrationAzure) error {
```
Comment (vijaykotechnologies) [inline azure.go:80]: AWS --> Azure

**Review by vijaykotechnologies — COMMENTED** [review]
A few minor comments made

**Comment by vijaykotechnologies** [comment]
Also update the README.md with 
Integrations with AWS
Integrations with Azure
BluePrint Definition

**Pull request merged**